package.json

```diff
@@ -1,34 +1,37 @@
 {
   "private": true,
   "repository": "git@github.com:Recidiviz/public-dashboard.git",
-  "workspaces": [
-    "public-dashboard-client",
-    "spotlight-api",
-    "spotlight-client"
-  ],
   "author": "Recidiviz <team@recidiviz.org>",
   "license": "GNU General Public License v3",
   "scripts": {
-    "demo:pd": "IS_DEMO=true REACT_APP_API_URL=http://localhost:3001 yarn dev:pd",
-    "dev:api": "yarn workspace spotlight-api run dev",
-    "dev:pd": "run-p dev:api dev:pdc",
-    "dev:pdc": "yarn workspace public-dashboard-client run dev",
-    "dev:sc": "yarn workspace spotlight-client run dev",
-    "demo:spotlight": "IS_DEMO=true REACT_APP_API_URL=http://localhost:3001 yarn dev:spotlight",
-    "dev:spotlight": "run-p dev:api dev:sc",
+    "dev:api": "cd spotlight-api && yarn dev",
+    "dev:sc": "cd spotlight-client && yarn dev",
+    "demo": "IS_DEMO=true REACT_APP_API_URL=http://localhost:3001 yarn dev",
+    "dev": "run-p dev:api dev:sc",
+    "install:api": "cd spotlight-api && yarn",
+    "install:sc": "cd spotlight-client && yarn",
+    "postinstall": "run-p install:*",
```

Comment:
Why is this called `postinstall`? Seems like `install:all` would make more sense?

Reply:
`postinstall` is an npm hook that will be automatically run after you run `yarn install` ... this way you don't have to remember to do it when you install at the root